aws/docker-pull.sh

```diff
@@ -15,6 +15,7 @@ IMAGES=(
   mozilla/syncserver
   mozilla/fxa-profile-server
   mozilla/pushbox
+  mozilla/pairsona
```

Comment:
@rfk Are we going to rename this at some point?

Reply:
I filed https://github.com/mozilla-services/pairsona/issues/26 to rename this, it will depend on the upstream repo publishing under correct name; /cc @jrgm which I believe has the right circleci privs to update to.